rich: add page

nik-sm opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

1.6.0 - https://github.com/Textualize/rich-cli

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=8044) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=8044) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=8044) it.</sub>

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Show file contents with syntax highlighting:
+
+`rich file.py`
```
```suggestion
`rich {{path/to/file.py}}`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Add line numbers, and indentation guides:
+
+`rich file.py --line-number --guides`
```
```suggestion
`rich {{path/to/file.py}} --line-number --guides`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Apply a theme:
+
+`rich file.py --theme monokai`
```
```suggestion
`rich {{path/to/file.py}} --theme {{monokai}}`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Display file in an interactive pager:
+
+`rich file.py --pager`
```
```suggestion
`rich {{path/to/file.py}} --pager`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Display contents from a URL:
+
+`rich https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md --markdown --pager`
```
```suggestion
`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Export a file as HTML:
+
+`rich file.md -o file.html`
```
```suggestion
`rich {{path/to/file.md}} -o {{path/to/file.html}}`
```

blueskyson commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Print text with formatting tags, custom alignment, and fixed line width:
+
+`rich --print "Hello [green on black]Stylized[/green on black] [bold]World[/bold]" --center --width 10`
```
```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`
```

blueskyson reviewed (commented):
Thank you for contribution! We use `{{ ... }}` to highlight user-provided token, here are some suggestions.

nik-sm commented:
Thanks for the feedback! Let me know if any other changes would be useful

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+> Rich CLI - fancy terminal display.
+> More information: <https://github.com/Textualize/rich-cli>.
+
+- Show file contents with syntax highlighting:
```
```suggestion
- Show a specific file with syntax highlighting:
```

EmilySeville7cfg replied:
@navarroaxel, what about to stick to `Display` word instead of `Print` in all pages?

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+`rich {{path/to/file.py}} --line-number --guides`
+
+- Apply a theme:
```
```suggestion
- Apply a specific theme:
```

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+`rich {{path/to/file.py}} --theme {{monokai}}`
+
+- Display file in an interactive pager:
```
```suggestion
- Display a specific file in an interactive pager:
```

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+`rich {{path/to/file.py}} --pager`
+
+- Display contents from a URL:
```
```suggestion
- Display contents from a specific URL:
```

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
+
+- Export a file as HTML:
```
```suggestion
- Export a specific file as HTML:
```

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+`rich {{path/to/file.md}} -o {{path/to/file.html}}`
+
+- Print text with formatting tags, custom alignment, and fixed line width:
```
```suggestion
- Display a text with specific formatting tags, alignment, and line width:
```

EmilySeville7cfg commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Print text with formatting tags, custom alignment, and fixed line width:
+
+`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`
```
```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --{{left|center|right}} --width {{10}}`
```

nik-sm commented:
Hi there - thanks for the feedback! I incorporated some of the suggested changes above.
I removed the word `specific` to follow the grammatical style of a few other sample pages (`tldr cat`, `tldr pip`, `tldr vim`) - let me know what you think.

navarroaxel commented on pages/common/rich.md:
```diff
@@ -0,0 +1,32 @@
+
+- Export a file as HTML:
+
+`rich {{path/to/file.md}} -o {{path/to/file.html}}`
```
We prefer long arguments if available
```suggestion
`rich {{path/to/file.md}} --export-html {{path/to/file.html}}`
```

EmilySeville7cfg commented:
Merged, thanks? 👍🏻
